GeoLib/Station.cpp

```diff
@@ -23,13 +23,13 @@
 
 namespace GeoLib
 {
-Station::Station(double x, double y, double z, std::string name) :
-    Point (x,y,z), _name(std::move(name)), _type(Station::StationType::STATION),
+Station::Station(double x, double y, double z, std::string const& name) :
```

Comment:
I am right now preparing a PR such that station names can be changed later (this is a request for the new GW-book). This might be interfering with that request?